### PR TITLE
Update each SDK readme with code examples

### PR DESCRIPTION
**Pull request by karansohi**
##### Checklist

- [ ] Tested in playground or other setup
- [ ] Screenshot (Grafana) from playground added to PR for 15+ minute run
- [x] Documentation is changed or added
- [ ] Tests and/or benchmarks are included
- [ ] Breaking changes

## Summary by CodeRabbit

- **Documentation**
	- Updated the README.md for the C# SDK with a new "Usage" section including code examples.
	- Revised the functionality description for the `aperture-go` SDK to emphasize integration and flow control.
	- Added usage instructions for the Java SDK, detailing client creation and flow functionality usage.
	- Enhanced the `aperture-js` SDK documentation for clarity, updated links, and expanded the usage section.
	- Improved the `aperture-py` SDK documentation with better integration instructions and added a new flow functionality example.

### CODE REVIEW

File: sdks/aperture-js/README.md [hunk 1]
```diff
@@ -14,9 +14,9 @@
 
 # Aperture JavaScript SDK
 
-`aperture-js` is an SDK to interact with [Aperture](https://docs.fluxninja.com)
-Agent. It allows flow control functionality on fine-grained features inside
-service code.
+The `aperture-js` SDK provides an easy to way to integrate your js applications
```
Comment (hdkshingala) [inline sdks/aperture-js/README.md:17]: There is a typo here `an easy to way to` -> `an easy way to`

File: sdks/aperture-csharp/README.md [hunk 2]
```diff
@@ -14,5 +14,71 @@
 
 # C# SDK for FluxNinja Aperture
 
-C# SDK provides an easy way to integrate your .NET applications with
+The C# SDK provides an easy way to integrate your .NET applications with
```
Comment (hdkshingala) [inline sdks/aperture-csharp/README.md:17]: Should we add `It allows flow
control functionality on fine-grained features inside service code.` in this and Java SDK similar to other SDKs?

Comment (karansohi) [reply sdks/aperture-csharp/README.md:17]: All SDKs provide same functionality.

File: sdks/aperture-js/README.md (hunk 1 above)
Comment (karansohi) [reply sdks/aperture-js/README.md:17]: Fixed in next commit.

File: sdks/aperture-csharp/README.md (hunk 2 above)
Comment (hdkshingala) [reply sdks/aperture-csharp/README.md:17]: Yes so we should have the same content in the README file of all SDKs, right?

**Review by hdkshingala — APPROVED** [review]

**Pull request merged**